crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -139,6 +144,11 @@ def adminrepocheck
           end
 
           unless zypper_stream["prompt"].nil?
+            upgrade_status.end_step(
+              false,
+              data: zypper_stream["prompt"]["text"],
+              help: "Make sure you solve the required action and try again."
```

Comment:
it might be any interactive prompt, not necessarily a problem i think, @skazi0 you ran into this once IIRC?